packages/web-components/src/stories/frameworks/react.stories.mdx

```diff
@@ -20,7 +20,7 @@ Otherwise, use:
 
 Astro Web Components make use of Stencil's automatic lazy loader which only loads components that are actually used on the page.
 
-Import the `@astrouxds/astro-web-components` main CSS file into your `index.js` or `app.js` file.
+Our dark theme component styles come out of the box, but if you need light theme or theme switching capabilities then include the following CSS import into your `index.js` or `app.js`:
```

Comment:
we also add some minor global styles like background color, the scrollbars, links, etc. so if you don't import the css, you have to do all of that yourself.

thats why ive been holding off on mentioning this bc we need to figure out an easy way for people to apply these globals themselves I think